submissions/homework_6/cc1759/hw.sql

```diff
@@ -0,0 +1,221 @@
+SET @last_memb_id = (SELECT LAST_INSERT_ID());
+
+-- add active checkouts
+-- should  result in (4, 1), (7, 1), (5, 2), (2, 4), (3, 4), (4, 4)
+INSERT INTO checkouts (book_id,  member_id, checkout_date, due_date, returned_date) VALUES
+    (@last_book_id + 3, @last_memb_id + 0, '2012-04-13 6:02:57', '2012-4-27', '2012-04-26 16:23:44'), 
+    (@last_book_id + 5, @last_memb_id + 0, DEFAULT, DEFAULT, NULL),
+    (@last_book_id + 4, @last_memb_id + 1, DEFAULT, DEFAULT, NULL), 
+    (@last_book_id + 1, @last_memb_id + 3, DEFAULT, DEFAULT, NULL), 
+    (@last_book_id + 2, @last_memb_id + 3, '2015-11-11 13:10:08', '2015-11-25', '2015-11-19 13:30:41'),
+    (@last_book_id + 0, @last_memb_id + 3, '2017-10-18 13:23:44', '2017-11-1', '2017-10-29 13:47:13');
+
+-- add web profiles
+-- should be (1, 6), 4, and 2
+INSERT INTO web_profiles (member_id, email, fav_book) VALUES
+    (@last_memb_id + 0, "emily-in-desguise@estupido.com", @last_book_id + 5),
+    (@last_memb_id + 3, "uncleweaven@gmail.com", NULL),
+    (@last_memb_id + 1, "mom@gmail.com", NULL);
+SET @last_prof_id = (SELECT LAST_INSERT_ID());
+
+-- add review
+-- should be (5, 1), (5, 2), (3, 2)
+INSERT INTO book_reviews (book_id, memb_web_prof_id, review, rating) VALUES
+    (@last_book_id + 4, @last_prof_id + 0, "OMG, litterally the BEST book EVER!!! CHANGED my WHOLE LIFE!!!! 10/10, wouldn't recomend, I would DAMAND you to read it!!!!!", 5),
+    (@last_book_id + 4, @last_prof_id + 1, "Teeming with Adventure", 3.7),
+    (@last_book_id + 3, @last_prof_id + 1, "Very Relatable", 4.8);
+
+
+
+-- SELECTING DATA
+
+-- 1. select all members
+SELECT * FROM members;
+
+-- 2. select all members full name and phone number
+SELECT full_name, phone_numb FROM members ORDER BY full_name ASC;
+
+-- 3. select all books published after 2010
+SELECT title, year_published FROM books 
+    WHERE year_published > 2010 
+    ORDER BY year_published ASC;
+
+-- 4. select the 3 most recent checkouts
+SELECT * FROM checkouts ORDER BY checkout_date DESC LIMIT 3;
+
+-- 5. select all Uncle Steavens checkouts
+SELECT * FROM checkouts WHERE member_id = @last_memb_id + 3;
+
+-- 6. select the title and isbn of all books with python in the title
+SELECT title, isbn FROM books WHERE title LIKE "%python%";
+
+-- 7. select all web profiles that have .edu emails
+SELECT * FROM web_profiles WHERE email LIKE "%.edu%";
+
+-- 8. select all books startign with the
+SELECT * FROM books WHERE title LIKE "the%";
+
+-- 9. select active checkouts
+SELECT * FROM checkouts WHERE returned_date = NULL;
+
+-- 10. select all members under an allias 'members' 
+SELECT full_name AS members FROM members;
+
+-- 11. select the age of each book
+-- (the next line didnt work for some reason)
+-- SELECT YEAR(CURRENT_DATE()) - year_published AS age FROM books;
+SELECT DATE_FORMAT(NOW(), '%Y') - year_published AS age FROM books
+    ORDER BY age DESC;
+
+-- 12. select the alphabeticly first two books
+SELECT * FROM books ORDER BY title ASC LIMIT 2;
```

Comment:
It might make sense to `LOWER` or `UPPER` the title if we're trying to sort alphabetically without case sensitivity.

```suggestion
SELECT * FROM books ORDER BY UPPER(title) ASC LIMIT 2;
```